tests/examples/marked_tracks.py

```diff
@@ -36,11 +35,24 @@ def test_mark_track3(self, es_cluster, rally):
 
 @pytest.mark.track("test-track")
 class TestMarkedClass:
-    @pytest.mark.track("test-track","test-track2")
     def test_mark_track(self, es_cluster, rally):
-        rally.race(track="test-track",challenge="index-only")
-        rally.race(track="test-track2",challenge="force-merge")
+        rally.race(track="test-track",challenge="force-merge")
 
     @pytest.mark.track("test-track3")
     def test_mark_track3(self, es_cluster, rally):
         rally.race(track="test-track3",challenge="index-only")
+
+class TestMarkedClassSubTrack:
+    @pytest.mark.track("many-tracks/sub-track")
+    def test_mark_sub_track(self, es_cluster, rally):
+        rally.race(track="many-tracks/sub-track",challenge="index-only")
+
+    @pytest.mark.track("many-tracks/sub-track2,test-track")
+    def test_mark_sub_track2(self, es_cluster, rally):
+        rally.race(track="many-tracks/sub-track2",challenge="index-only")
+        rally.race(track="test-track",challenge="index-only")
```

Comment:
Nit: it's not clear why `test-track` is used in the marker and as a track if the name of this class is `...SubTrack`.